xpcPlugin/DataManager.cpp

```diff
@@ -382,7 +382,7 @@ namespace XPC
 		}
 		if ((dataType & 32) == 32) // Byte array
 		{
-			const std::size_t TMP_SIZE = 1024;
+			const std::size_t TMP_SIZE = XPC_MAX_DREF_VALUES;
```

Comment:
strange to have a different cap of 1024 here, since requests ask for max `size` of 255 values (so count fits in one byte), so even if we find 1024 values we then discard down to 255 anyway